canvas_oauth/canvas.py

```diff
@@ -20,12 +20,15 @@ def get_oauth_login_url(client_id, redirect_uri, response_type='code',
     """
     authorize_url = AUTHORIZE_URL_PATTERN % settings.CANVAS_OAUTH_CANVAS_DOMAIN
 
+    if scopes and not isinstance(scopes, str):
```

Comment:
Why this particular check, and a silent failure if it isn't met?

Reply:
Great question. The intent is to allow the caller to provide the `scopes` parameter as either a string or a list, and convert to a string in the case it's a non-empty list. But perhaps that would be clearer if it was just one or the other, eliminating the need for this logic.